jax/experimental/array_api/_statistical_functions.py

```diff
@@ -13,47 +13,14 @@
 # limitations under the License.
 
 import jax
-from jax.experimental.array_api._data_type_functions import (
-    _promote_to_default_dtype,
-)
-
-
-def cumulative_sum(x, /, *, axis=None, dtype=None, include_initial=False):
-  """Calculates the cumulative sum of elements in the input array x."""
-  return jax.numpy.cumulative_sum(x, axis=axis, dtype=dtype, include_initial=include_initial)
-
-def max(x, /, *, axis=None, keepdims=False):
-  """Calculates the maximum value of the input array x."""
-  return jax.numpy.max(x, axis=axis, keepdims=keepdims)
-
-
-def mean(x, /, *, axis=None, keepdims=False):
-  """Calculates the arithmetic mean of the input array x."""
-  return jax.numpy.mean(x, axis=axis, keepdims=keepdims)
-
-
-def min(x, /, *, axis=None, keepdims=False):
-  """Calculates the minimum value of the input array x."""
-  return jax.numpy.min(x, axis=axis, keepdims=keepdims)
-
-
-def prod(x, /, *, axis=None, dtype=None, keepdims=False):
-  """Calculates the product of input array x elements."""
-  x = _promote_to_default_dtype(x)
-  return jax.numpy.prod(x, axis=axis, dtype=dtype, keepdims=keepdims)
-
 
+# TODO(micky774): Remove after updating signatures to take array via
+# positional-only argument
```

Comment:
I don't think we need this, because allowing `x` as a keyword is a superset of the behavior specified by the array API.